candle/my_timetable/my_timetable.py

```diff
@@ -22,11 +30,32 @@ def get_timetable(id_):
     id_ = int(id_)
     return current_user.timetables.filter(UserTimetable.id_ == id_).first_or_404()
 
+
+def get_highlighted_lesson_ids(user_timetable_id: int) -> Set[int]:
+    highlighted_user_timetable_lessons = db.session.query(
+        user_timetable_lessons,
+    ).filter(
+        user_timetable_lessons.c.user_timetable_id == user_timetable_id,
+        user_timetable_lessons.c.highlighted == True
+    ).all()
+    return {
+        item._mapping["lesson_id"]
+        for item in highlighted_user_timetable_lessons
+    }
+
+
 @my_timetable.route('/moj-rozvrh/<id_>')
 @login_required
 def show_timetable(id_):
     ut = get_timetable(id_)
-    return render_timetable(ut.name, ut.lessons, selected_timetable_key=int(id_))
+
+    return render_timetable(
+        title=ut.name,
+        lessons=ut.lessons,
+        highlighted_lesson_ids=get_highlighted_lesson_ids(ut.id_),
+        selected_timetable_key=int(id_),
+        render_lesson_actions=True,
```

Comment:
Neznamená `render_lesson_actions` to isté ako `editable`?

Reply:
Good catch, ocividne znamena :) Fixnute